### ISSUE TITLE
Report error message when start

### ISSUE THREAD

**Issue by tan-wei** [nxvu699134/vn-night.nvim#2]
```
处理 ColorScheme 自动命令 "*" 时发生错误: 
E121: Undefined variable: g:colors_name 
E116: Invalid arguments for function CocHelperCallFn_m1_v0_13_0
请按 ENTER 或其它命令继续
```

It seems something with `coc`.

**Comment by nxvu699134** [comment]
Is it working with COC now @tan-wei.

**Comment by tan-wei** [comment]
Sorry for late reply. I'll try it ASAP.

**Comment by tan-wei** [comment]
No, it still complains the same error. Neovim version: `v0.8.0-dev-1112-g6d557e324`

**Comment by tan-wei** [comment]
OK, I think I find the issue:
  - I use a plugin which random choice a theme from a set, the plugin will print the name of the chosen colorscheme by echo `g:color_names`
  - But `vn-night.nvim` does not set the variable

It is easy to reproduce without the plugin:
  - Just print the name of current colorscheme by intputing command `:colorscheme`, neovim will just print `default` instead of `vn-night`

So set `g:color_names` will make this issue closed.

**Comment by tan-wei** [comment]
For any other colorschemes, the variable is set properly, so should `vn-night.nvim`, is it?

[For example]( https://github.com/rockerBOO/boo-colorscheme-nvim/blob/a8659f879cfd23c05b1104e916f879bcd628c5ec/lua/boo-colorscheme.lua#L798C2-L798C2 )

**Comment by tan-wei** [comment]
Great!
